app/models/order.rb

```diff
@@ -18,4 +18,10 @@ def self.sort_status
   def total_item_quantity
     item_orders.sum(:quantity)
   end
+
+  def status_to_packaged
```

Comment:
this method will probably be called when the merchant updates orders in later user stories